pkg/repo/chartrepo.go

```diff
@@ -285,6 +285,7 @@ func ResolveReferenceURL(baseURL, refURL string) (string, error) {
 	}
 
 	// We need a trailing slash for ResolveReference to work, but make sure there isn't already one
+	parsedBaseURL.RawPath = strings.TrimSuffix(parsedBaseURL.RawPath, "/") + "/"
 	parsedBaseURL.Path = strings.TrimSuffix(parsedBaseURL.Path, "/") + "/"
```

Comment:
```suggestion
```

I would prefer to remove this and change line 277.
```golang
// We need a trailing slash for ResolveReference to work, but make sure there isn't already one
	parsedBaseURL, err := url.Parse(strings.TrimSuffix(baseURL, "/") + "/"
```

Reply:
Good idea @cndoit18. Done.